plugins/grafana-sankey-plugin/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## 1.0.0 (Unreleased)
+
+Initial release.
```

Comment:
A changelog for the plugin might make sense, but in that case releases should be coordinated with Antrea releases; it doesn't seem we're releasing the plugin independently from Antrea.

Reply:
I think for now we can upload the plugin to Antrea S3 storage bucket. I've started to talk to Beijing team for the S3 access permission. So that the plugin update will not be coupled with Antrea releases. 
I kept the CHANGELOG and removed the release-assets-upload github workflow.